src/main/java/cat/udl/eps/softarch/learnphysics/repository/LevelRepository.java

```diff
@@ -1,9 +1,17 @@
 package cat.udl.eps.softarch.learnphysics.repository;
 
 import cat.udl.eps.softarch.learnphysics.domain.Level;
+import cat.udl.eps.softarch.learnphysics.domain.Topic;
 import org.springframework.data.repository.PagingAndSortingRepository;
-import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+import org.springframework.data.repository.query.Param;
+
+import java.util.Optional;
 
-@RepositoryRestResource
```

Comment:
Without this annotation, the endpoint `/levels` won't be exposed, so you won't be able to test it.